capability/capability_linux.go

```diff
@@ -100,12 +98,7 @@ func mkString(c Capabilities, max CapType) (ret string) {
 
 func newPid(pid int) (c Capabilities, err error) {
 	switch capVers {
-	case linuxCapVer1:
-		p := new(capsV1)
-		p.hdr.version = capVers
-		p.hdr.pid = int32(pid)
-		c = p
-	case linuxCapVer2, linuxCapVer3:
+	case linuxCapVer3:
```

Comment:
If capVers is unknown version, we need to fall back to linuxCapVer3. Otherwise everything will be broken when a new version is introduced.

Reply:
👍🏻 Makes sense, please see updated commits.